src/aave-v3/SupplyVault.sol

```diff
@@ -215,15 +215,15 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
             uint256 claimedAmount = claimedAmounts[i];
             uint256 rewardsIndexMem = rewardsIndex[rewardToken];
 
-            if (supply > 0 && claimedAmount > 0) {
+            if (claimedAmount > 0) {
```

Comment:
Why this change ? If the total supply is 0 because of withdraws we want to be able to accrue rewards. Even though I understand that this should never happen in practice, I think we said that we want to consider this possibility

Reply:
The condition removed here is actually covering the edge-case you are mentioning (we are loosening the condition here)
But the rationale behind this change was the redundancy check `totalSupply > 0` performed twice (once here, and once inside `_getUnaccruedRewardIndex`)

Reply:
Ok sounds good to me !